torch/distributed/rpc/__init__.py

```diff
@@ -53,7 +53,10 @@ def init_model_parallel(
             self_rank (int): a globally unique id/rank of this node.
             init_method(str): backend specific init arguments.
             num_send_recv_threads(int): Number of threads for send/recv work.
-            rpc_timeout (datetime.timedelta): Timeout for RPCs. Defaults to 10 seconds.
+            rpc_timeout (datetime.timedelta):
+                Global timeout for server to process an RPC request.
```

Comment:
Before we made the decision on whether we should have finer grain timeout (server, client, etc.), shall we avoid mentioning server here?

Reply:
Makes sense. Removing it for now.